tests/test_report_manager.py

```diff
@@ -44,6 +53,29 @@ def _evaluate_config(self, args, yaml_content):
         mock_config.stop()
         return config
 
+    def _setUp_managers(self,
```

Comment:
If you have snake_case, you don't want to mix in camelCase. You can change this to `_init_managers`

Reply:
Changed